<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDelimitedTextPluginGuiBase</class>
 <widget class="QDialog" name="QgsDelimitedTextPluginGuiBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a Layer from a Delimited Text File</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delimited text file</string>
       </property>
       <property name="buddy">
        <cstring>txtFilePath</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsFileDropEdit" name="txtFilePath">
       <property name="toolTip">
        <string>Full path to the delimited text file</string>
       </property>
       <property name="whatsThis">
        <string>Full path to the delimited text file. In order to properly parse the fields in the file, the delimiter must be defined prior to entering the file name. Use the Browse button to the right of this field to choose the input file.</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnBrowseForFile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Browse to find the delimited text file to be processed</string>
       </property>
       <property name="whatsThis">
        <string>Use this button to browse to the location of the delimited text file. This button will not be enabled until a delimiter has been entered in the &lt;i&gt;Delimiter&lt;/i&gt; box. Once a file is chosen, the X and Y field drop-down boxes will be populated with the fields from the delimited text file.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel1_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Layer name</string>
       </property>
       <property name="buddy">
        <cstring>txtLayerName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtLayerName">
       <property name="toolTip">
        <string>Name to display in the map legend</string>
       </property>
       <property name="whatsThis">
        <string>Name displayed in the map legend</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Delimiter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Delimiter string</string>
        </property>
        <property name="buddy">
         <cstring>txtDelimiter</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtDelimiter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Delimiter to use when splitting fields in the text file. The delimiter can be more than one character.</string>
        </property>
        <property name="whatsThis">
         <string>Delimiter to use when splitting fields in the delimited text file. The delimiter can be 1 or more characters in length.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>delimiterPlain</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="delimiterPlain">
        <property name="toolTip">
         <string>The delimiter is taken as is</string>
        </property>
        <property name="whatsThis">
         <string>The delimiter is taken as is</string>
        </property>
        <property name="text">
         <string>Plain characters</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="delimiterRegexp">
        <property name="toolTip">
         <string>The delimiter is a regular expression</string>
        </property>
        <property name="whatsThis">
         <string>The delimiter is a regular expression</string>
        </property>
        <property name="text">
         <string>Regular expression</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Geometry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>&lt;p align=&quot;right&quot;&gt;X field&lt;/p&gt;</string>
        </property>
        <property name="buddy">
         <cstring>cmbXField</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbXField">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Name of the field containing x values</string>
        </property>
        <property name="whatsThis">
         <string>Name of the field containing x values. Choose a field from the list. The list is generated by parsing the header row of the delimited text file.</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="cmbYField">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Name of the field containing y values</string>
        </property>
        <property name="whatsThis">
         <string>Name of the field containing y values. Choose a field from the list. The list is generated by parsing the header row of the delimited text file.</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="textLabel2_2">
        <property name="text">
         <string>&lt;p align=&quot;right&quot;&gt;Y field&lt;/p&gt;</string>
        </property>
        <property name="buddy">
         <cstring>cmbYField</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sample text</string>
     </property>
     <property name="buddy">
      <cstring>txtSample</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtSample">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsFileDropEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfiledropedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtFilePath</tabstop>
  <tabstop>btnBrowseForFile</tabstop>
  <tabstop>txtLayerName</tabstop>
  <tabstop>txtDelimiter</tabstop>
  <tabstop>delimiterPlain</tabstop>
  <tabstop>delimiterRegexp</tabstop>
  <tabstop>cmbXField</tabstop>
  <tabstop>cmbYField</tabstop>
  <tabstop>txtSample</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="delimited_text.qrc"/>
 </resources>
 <connections/>
</ui>
